packages/client/package.json

```diff
@@ -3,9 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.2.1",
+    "@fortawesome/free-brands-svg-icons": "^6.2.1",
+    "@fortawesome/free-regular-svg-icons": "^6.2.1",
+    "@fortawesome/free-solid-svg-icons": "^6.2.1",
+    "@fortawesome/react-fontawesome": "^0.2.0",
     "@testing-library/jest-dom": "^5.16.4",
```

Comment:
No changes to the dependencies that are not agreed upon with everybody. 
Icons are in the design system, and should be placed in /public and used from there. 
Fix that, then I can review the rest. 🙏🏽

Reply:
ok, will remove fontawesome; but maybe we can keep that as I already did it?

Reply:
no 😅 
sorry 🙏🏽